website_no_crawler/views/disable_robots.xml

```diff
@@ -1,8 +1,10 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <odoo>
-    <template id="website.robots">
+    <template id="robots" inherit_id="website.robots" primary="True">
```

Comment:
Please solve this discussion :smile:

Reply:
Yes sorry. AFAIK primary serves for the purpose of replacing a view, so `website.robots` now resolves to this view instead of the other as the base.

This way, when the module is uninstalled and this `website_no_crawler.robots` view is removed, `website.robots` recovers its primary status and works as before.

At least, that's the purpose of this patch.

Reply:
Then you must ensure that the priority of this view is smaller than the other one, or it won't "resolve" to this one. Any way, are you sure the XML-ID is not hardcoded and this one it's in fact used?

Reply:
> are you sure the XML-ID is not hardcoded and this one it's in fact used?

Since v11, the important part is the `key` of the view, not the XMLID.

Anyway, I'm rebuilding the runbot. I think it's better to test this before continuing :blush:

Reply:
You were right! This wasn't working properly.

It turns out you can replace text nodes and, to avoid a `<p>` tag automatically being printed, you have to wrap it in a `<t>` tag.

Well, it's working now. And it uninstalls properly.